config/locales/en.yml

```diff
@@ -545,7 +545,7 @@ en:
         sample_rate: Frequency
         status: Status
       new:
-        hint: Upload a file
+        hint: Upload a file or drag it here
```

Comment:
Figure some language to indicate drag n drop is available might be enough in and of itself.